blueprints-1.16.0: Error: Cannot find module '../../lib/utils/ipv6-utils'

### Describe the bug

I followed the instructions in the README to setup a new project both on my workstation and in a fresh node container, each resulted in the same error.

### Expected Behavior

CDK deploys stack

### Current Behavior

```
root@2c8bcc199fe3:/my-blueprints# cdk deploy 
Error: Cannot find module '../../lib/utils/ipv6-utils'
Require stack:
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/resource-providers/ipv6-iam.js
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/resource-providers/index.js
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/cluster-providers/import-cluster-provider.js
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/cluster-providers/index.js
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/addons/appmesh/index.js
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/addons/index.js
- /my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/index.js
- /my-blueprints/bin/my-blueprints.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/my-blueprints/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/lib/resource-providers/ipv6-iam.ts:4:1)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Object.require.extensions.<computed> [as .js] (/my-blueprints/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:1288:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/resource-providers/ipv6-iam.js',
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/resource-providers/index.js',
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/cluster-providers/import-cluster-provider.js',
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/cluster-providers/index.js',
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/addons/appmesh/index.js',
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/addons/index.js',
    '/my-blueprints/node_modules/@aws-quickstart/eks-blueprints/dist/index.js',
    '/my-blueprints/bin/my-blueprints.ts'
  ]
}
Subprocess exited with error 1
```

### Reproduction Steps

https://github.com/aws-quickstart/cdk-eks-blueprints?tab=readme-ov-file#getting-started-1

### Possible Solution

_No response_

### Additional Information/Context

_No response_

### CDK CLI Version

2.162.1

### EKS Blueprints Version

1.16.0

### Node.js Version

v20.18.0

### Environment details (OS name and version, etc.)

Debian 12

### Other information

_No response_

--- a/lib/resource-providers/ipv6-iam.ts
+++ b/lib/resource-providers/ipv6-iam.ts
@@ -1,7 +1,7 @@
 import * as spi from "../spi";
 import * as iam from "aws-cdk-lib/aws-iam";
 import {IManagedPolicy} from "aws-cdk-lib/aws-iam";
-import {getEKSNodeIpv6PolicyDocument} from '../../lib/utils/ipv6-utils';
+import {getEKSNodeIpv6PolicyDocument} from '../utils/ipv6-utils';
 
 /**
  * Resource provider that creates a new role with ipv6 permissions.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "@aws-quickstart/eks-blueprints",
-    "version": "1.16.0",
+    "version": "1.16.1",
     "license": "Apache-2.0",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",

